bakery/RB_1qb_baking_2.py

```diff
@@ -0,0 +1,88 @@
+from qm import SimulationConfig
+from qm.QmJob import QmJob
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from RB_1qb_configuration import config, pulse_len
+from rb_1_with_table_guide import *
```

Comment:
please avoid import * here because it's hard to know what is being used from there (makes the code less readable)

Reply:
Import is now more explicit